content/docs/troubleshooting/_index.md

```diff
@@ -280,3 +280,59 @@ field once the Ingress resource is ready to route traffic.
 For the Traefik controller, verify that the `kubernetes.ingressEndpoint` config
 is [set properly](https://docs.traefik.io/providers/kubernetes-ingress/). This option was
 introduced in Traefik 1.7.0.
+
+## Synchronous call was made to "X" with an unregistered Provider. {#synchronous-call}
+
+The warning occurs when making a 'data source' call in Pulumi in a synchronous fashion and passing along a ProivderResource that has not been registered. For example:
+
+```ts
+const provider = new aws.Provider(...);
+
+// A call to some provider's `getXXX` data source function.
+const ids = aws.ec2.getSubnetIds(..., { provider });
+
+// or
+
+const parent = new SomeResource("name", { provider });
+const ids = aws.ec2.getSubnetIds(..., { parent });
+```
+
+This warning may be benign. However, if you are experiencing crashes or hangs in Pulumi (especially in Node.js version 12.0 and above)
```

Comment:
```suggestion
This warning may be benign. However, if you are experiencing crashes or hangs in Pulumi (especially in Node.js version 12.11.0 and above)
```

^^^ isn't this when it technically began? Being precise here will help with searchability.